include/swift/SIL/SILCloner.h

```diff
@@ -1409,14 +1409,12 @@ SILCloner<ImplClass>::visitDebugValueInst(DebugValueInst *Inst) {
     return;
 
   // Since we want the debug info to survive, we do not remap the location here.
-  SILDebugVariable VarInfo = *Inst->getVarInfo();
+  auto VarInfo = Inst->getVarInfo();
```

Comment:
It's LLVM style to only use `auto` if the type is obvious from the function name (e.g: auto *insn = dyn_cast<SILInstruction>(ptr)`. Especially `auto` without a reference or pointer is suspicious because it looks like we may copy a type.